src/main/java/bio/terra/service/dataset/DatasetRequestValidator.java

```diff
@@ -12,15 +12,13 @@
 import bio.terra.model.DatasetSpecificationModel;
 import bio.terra.model.TableModel;
 import bio.terra.common.ValidationUtils;
+import com.google.common.collect.ImmutableSet;
 import org.springframework.stereotype.Component;
 import org.springframework.validation.Errors;
 import org.springframework.validation.Validator;
 
 import javax.validation.constraints.NotNull;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Optional;
+import java.util.*;
```

Comment:
The jade/terra code follows google code conventions, so wildcard imports aren't used. I believe you can configure intellij so it doesn't use wildcards. https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports